<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>866</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="displayExperimentPath"/>
      </item>
      <item>
       <widget class="QPushButton" name="selectExperimentButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Select experiment</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>22</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="displayTheoryPath"/>
      </item>
      <item>
       <widget class="QPushButton" name="selectTheoryButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Select theory</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="mplWindow" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="mplvl"/>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>135</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>135</width>
            <height>60</height>
           </size>
          </property>
          <property name="title">
           <string>Magnetic field fit</string>
          </property>
          <widget class="QLineEdit" name="minB">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>41</width>
             <height>25</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="maxB">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>30</y>
             <width>41</width>
             <height>25</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bestFitButton">
          <property name="text">
           <string>Fit all</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>135</width>
            <height>180</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>135</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>1 Parallel</string>
          </property>
          <widget class="QDoubleSpinBox" name="bParSpinBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>121</width>
             <height>26</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="yMaxParSpinBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>120</y>
             <width>121</width>
             <height>26</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="QLabel" name="label_13">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>51</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>y_max</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_14">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>16</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>b</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="bParStep">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>70</y>
             <width>21</width>
             <height>25</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="yMaxParStep">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>150</y>
             <width>21</width>
             <height>25</height>
            </rect>
           </property>
          </widget>
          <widget class="QCheckBox" name="fixedbParCheckBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>61</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Fixed</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="fixedymaxParCheckBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>150</y>
             <width>61</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Fixed</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>135</width>
            <height>180</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>2 Perpendicular</string>
          </property>
          <widget class="QDoubleSpinBox" name="bPerSpinBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>121</width>
             <height>26</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="yMaxPerSpinBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>120</y>
             <width>121</width>
             <height>26</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>y_max</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>21</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>b</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="bPerStep">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>70</y>
             <width>21</width>
             <height>25</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="yMaxPerStep">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>150</y>
             <width>21</width>
             <height>25</height>
            </rect>
           </property>
          </widget>
          <widget class="QCheckBox" name="fixedbPerCheckBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>61</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Fixed</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="fixedymaxPerCheckBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>150</y>
             <width>61</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Fixed</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>135</width>
            <height>250</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>3 Difference</string>
          </property>
          <widget class="QDoubleSpinBox" name="cSpinBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>91</width>
             <height>26</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="c1SpinBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>91</width>
             <height>26</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>21</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>c1</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>21</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>c</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="cStep">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>40</y>
             <width>21</width>
             <height>25</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="c1Step">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>100</y>
             <width>21</width>
             <height>25</height>
            </rect>
           </property>
          </widget>
          <widget class="QCheckBox" name="smoothDiffCheckBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>111</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Smooth Diff</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QCheckBox" name="smoothCircCheckBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>180</y>
             <width>111</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Smooth Circ</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QCheckBox" name="fixedCcheckBox">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>20</y>
             <width>61</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Fixed</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="fixedC1checkBox">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>80</y>
             <width>61</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Fixed</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="minDiff">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>150</y>
             <width>51</width>
             <height>25</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="maxDiff">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>150</y>
             <width>51</width>
             <height>25</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="minCirc">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>200</y>
             <width>51</width>
             <height>25</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_15">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>150</y>
             <width>16</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>y</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_16">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>200</y>
             <width>21</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>y</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="maxCirc">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>200</y>
             <width>51</width>
             <height>25</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="setButton">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>225</y>
             <width>41</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>928</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
